src/BackgroundTasks/Worker.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace ILIAS\BackgroundTasks;
+
+/**
+ * Interface Worker
+ *
+ * @package ILIAS\BackgroundTasks
+ */
+interface Worker {
+
+	/**
+	 * @return void
+	 */
+	public function doWork();
+
+	/**
+	 * Returns true iff the worker wants to be called within the current HTTP request.
```

Comment:
@chfsx Did not know about this, I thought it was a well known abbreviation. I'd prefer if we could use it as it's extremly comfy to use. Iff that won't be the case i'll be sad.